### PR TITLE
Tighten CSP by using nonces for inline scripts, move survey page outside of the service, add initial payment mop-up package

### PR DESCRIPTION
**Pull request by graham-willis-druid**
Add survey page outside of service. This requires an environment variable to set up the location of the survey. See README. (I will slack you the link)

Add uri locations to the layout template for all requests using onPreResponse handler.

Include basic package for the mop up job with a start script and unit tests.

Tighten up CSP to remove all inline

### CODE REVIEW

File: packages/payment-mop-up-job/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@defra-fish/payment-mop-up-job",
+  "version": "0.0.1-beta.00",
```
Comment (sgardnerdell-ea) [inline packages/payment-mop-up-job/package.json:3]: Should be `0.0.1-beta.0` (not double-0)

File: packages/payment-mop-up-job/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "description": "Process incomplete web-sales",
+  "type": "module",
+  "engines": {
+    "node": ">=14"
+  },
+  "keywords": [
+    "rod",
+    "licensing",
+    "sales",
```
Comment (sgardnerdell-ea) [inline packages/payment-mop-up-job/package.json:12]: Not the sales api 😉

File: packages/payment-mop-up-job/package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "lib": "src",
+    "test": "__tests__"
+  },
+  "files": [
+    "src"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/DEFRA/rod-licensing.git",
+    "directory": "packages/pocl-job"
```
Comment (sgardnerdell-ea) [inline packages/payment-mop-up-job/package.json:28]: Not the pocl-job 😉

File: packages/payment-mop-up-job/package.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@defra-fish/payment-mop-up-job",
```
Comment (sgardnerdell-ea) [inline packages/payment-mop-up-job/package.json:2]: Can you update the link and text in the main project repo README.md as this differs from the name there

File: packages/gafl-webapp-service/src/server.js [hunk 5]
```diff
@@ -63,8 +70,8 @@ const plugIns = [
        * 31 of the GDS template. This will probably come up as an advisory in the PEN test.
        */
       fontSrc: ['self', 'fonts.gstatic.com', 'data:'],
-      scriptSrc: ['self', 'unsafe-inline'],
-      generateNonces: false
+      scriptSrc: ["'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='"],
```
Comment (sgardnerdell-ea) [inline packages/gafl-webapp-service/src/server.js:73]: Can we extract this to a descriptive variable name pls?

File: packages/gafl-webapp-service/src/server.js [hunk 6]
```diff
@@ -63,8 +70,8 @@ const plugIns = [
        * 31 of the GDS template. This will probably come up as an advisory in the PEN test.
```
Comment (sgardnerdell-ea) [inline packages/gafl-webapp-service/src/server.js:70]: Probably don't need this comment about PEN test anymore.

File: packages/payment-mop-up-job/package.json (hunk 1 above)
Comment (graham-willis-druid) [reply packages/payment-mop-up-job/package.json]: fixed

File: packages/payment-mop-up-job/package.json (hunk 2 above)
Comment (graham-willis-druid) [reply packages/payment-mop-up-job/package.json]: fixed

File: packages/payment-mop-up-job/package.json (hunk 4 above)
Comment (graham-willis-druid) [reply packages/payment-mop-up-job/package.json:2]: fixed

File: packages/gafl-webapp-service/src/server.js (hunk 5 above)
Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/server.js]: removed

File: packages/gafl-webapp-service/src/server.js (hunk 6 above)
Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/server.js]: removed

**Pull request merged**